node/src/service/mod.rs

```diff
@@ -212,11 +206,34 @@ where
 		&task_manager,
 	)?;
 	// Frontier stuffs.
-	let frontier_backend = Arc::new(fc_db::Backend::open(
-		Arc::clone(&client),
-		&config.database,
-		&crate::frontier_service::db_config_dir(config),
-	)?);
+	let overrides = fc_storage::overrides_handle(client.clone());
+	let db_config_dir = crate::frontier_service::db_config_dir(config);
+	let frontier_backend = match eth_rpc_config.frontier_backend_type {
```

Comment:
Can we move this to `frontier_service`?

Reply:
It's possible. Nice advice. I'll fix it.

Reply:
Open another PR for this.